package.json

```diff
@@ -20,6 +20,7 @@
     "grunt-contrib-copy": "^0.8.0",
     "grunt-contrib-jshint": "0.6.5",
     "grunt-contrib-uglify": "0.6.0",
+    "grunt-gh-pages": "^0.10.0",
```

Comment:
very cool. I remember looking for once but they were missing features, can't remember now.